src/networks.json

```
@@ -532,12 +532,16 @@
     "name": "Thundercore Mainnet",
     "chainId": 108,
     "network": "mainnet",
-    "multicall": "",
+    "multicall": "0x3017086deef56679e267f67f66c4415109b7a97f",
     "rpc": [
-      "https://mainnet-rpc.thundercore.com"
+      "https://mainnet-rpc.thundercore.com/archived/SNAPSHOTEuR82a75fLYA",
+      "https://mainnet-rpc.thundercore.com",
+      "https://mainnet-rpc.thundertoken.net",
+      "https://mainnet-rpc.thundercore.io"
```

```suggestion
      "https://mainnet-rpc.thundercore.com/archived/SNAPSHOTEuR82a75fLYA"
```
Only the first node seems to be full archive, we can remove others.